ictv-webapp

```diff
@@ -63,5 +63,6 @@ if __name__ == '__main__':
     if database.database_path is None:
         database.database_path = config['database_uri']
 
+    config['address_port'] = args.address_port
     update_database()
-    main(config_file)
+    main(config)
```

Comment:
We give the address and the port in argument to launch the server, at least it was like that before. We could add it in the config but I did it like this so that is the same command as before to launch the server

Reply:
To briefly explain, web.py automatically reads the first argument in the command line and set it as the address and port ([as explained here](https://github.com/webpy/webpy/blob/04bf405dd86485c8f3774c526e6a869a1eb73408/web/application.py#L352-L355)).

Flask doesn't do such thing, so I put it in the config to parse it later in app.py (Line 405). 

In fact, with web.py you can also only specify the port, that's what I fixed in my last commit